task/mgmt-gateway/src/mgs_handler.rs

```diff
@@ -152,4 +221,104 @@ impl SpHandler for MgsHandler {
             Err(ResponseError::Busy)
         }
     }
+
+    fn sys_reset_prepare(
```

Comment:
Nit: this name is slightly confusing, because functions beginning with `sys_` are normally syscalls (same for `sys_reset_trigger` below).

Reply:
Will change; I'm trying to remember why I added the `sys_` prefix in the first place. This is indirectly triggering a system call (made by jefe); maybe that was it.

Reply:
This requires a corresponding change in omicron, where this trait is defined. I'm going to defer changing this name on this PR, but will include it in the next PR that bumps the omicron dep.